### ISSUE TITLE
Add Fixer executable option

### ISSUE THREAD

**Issue by sicom-dn** [ObliviousHarmony/vscode-php-codesniffer#35]
### Problem Description

We have phpCodeSniffer.executable option where we can add phpcs executable path, but there is no way to define path to phpcbf, so if its located at non-standard path fixing doesnt work.

### Proposed Solution

Add phpCodeSniffer.fixerExecutable or something like this to define path to phpcbf

**Comment by ObliviousHarmony** [comment]
Hey @sicom-dn,

This extension only uses the `phpcs` executable. [These reports](https://github.com/ObliviousHarmony/vscode-php-codesniffer/tree/trunk/phpcs-reports) are used to generate the necessary data for VS Code to make the actual transformations. Are you experiencing some kind of issue?

**Comment by sicom-dn** [comment]
Hmm.. yes, @ObliviousHarmony there is an issue i thought related to phpcbf. I was trying to setup plugin under windows with phpcs installed globally. To make it work i had to provide path to phpcs but i still do not have option to fix errors. Any idea how  it can be fixed?

<img width="764" alt="CleanShot 2021-09-25 at 12 00 29@2x" src="https://user-images.githubusercontent.com/46018298/134766026-cccbaf64-1491-4835-a9c3-a0bcfb8af534.png">

**Comment by ObliviousHarmony** [comment]
@sicom-dn what happens when you hover over one of the problems in the text editor? Additionally, do you have access to the formatter to try that option too?

Lastly, see if there is an output channel for the extension with errors.

**Comment by sicom-dn** [comment]
``@ObliviousHarmony when i hover in text editor i get same option, ignore error. If i run phpcfb it does the fixes. Nothing in output channel.

However i just noticed even more strange error. It seems like phpcs in my setup is getting running once or its results are getting cached somehow (is it possible)? Thing is that doesnt matter what file i open it always shows me same report valid  for the file i initially tested on. Changing config, restarting VSCode or Windows doesnt seems affect anything... I wonder how its possible?

Here is what i have in workspace config related to phpCodeSniffer.
```
	"settings": {
		"phpCodeSniffer.standard": "Custom",
		"phpCodeSniffer.standardCustom": "C:\\Users\\42\\Local Sites\\test\\app\\public\\phpcs.xml",
		"phpCodeSniffer.executable": "C:\\Users\\42\\scoop\\apps\\composer\\current\\home\\vendor\\bin\\phpcs.bat",	
	}
```

**Comment by ObliviousHarmony** [comment]
Sorry for the delay @sicom-dn, could you give me some information about your configuration so I can try to figure this out? Are you using WSL, for instance? I've only ever tried this extension on Windows using it, so it's possible that I need to make some fixes for a native Windows execution context.

**Comment by sicom-dn** [comment]
well, i tried this setup first on Windows 10 running under VirtualBox and then validated it on notebook with Windows 10. In both cases i used [Scoop](https://scoop.sh/) package manager to install php 7.4, composer. I installed phpcs with composer globally (as it described on phpcs site). Generally i did following:

- `scoop bucket add php` 
- `scoop install php7.4`
- In your windows user dir open `scoop\apps\php7.4\current\php.ini` and uncommented `;extension_dir = "ext"` (line ~760) enabled modules bz2, curl, fileinfo, gettext, intl, mbstring, openssl (around line 910)
- `scoop install composer`
- `composer global require "squizlabs/php_codesniffer=*"`

**Comment by ObliviousHarmony** [comment]
This _may_ have been fixed in https://github.com/ObliviousHarmony/vscode-php-codesniffer/pull/39. I've just released a 1.5.0 version containing the fix, could you try @sicom-dn and see if that works for you?

**Comment by ObliviousHarmony** [comment]
I'm going to close this as stale, but let me know if you're still encountering a problem!
